source/atlas-cli-deploy-docker.txt

```diff
@@ -110,23 +110,27 @@ deployment with `Docker Compose <https://docs.docker.com/compose/>`__.
 
    .. step:: Create a ``docker-compose.yaml`` file.
 
-      Create the ``docker-compose.yaml`` file in the same directory 
-      that you run Docker Compose from.
+      a. Create the ``docker-compose.yaml`` file in the same directory 
+         that you run Docker Compose from.
 
-      **Example:**
+         **Example:**
 
-      .. code-block:: sh
+         .. code-block:: sh
+            :linenos: 
          
-         services:
-           mongo:
-             image: mongodb/atlas
-             privileged: true
-             command: |
-               /bin/bash -c "atlas deployments setup --type local --port 27778 --bindIpAll --username root --password root --force && tail -f /dev/null"
-             volumes:
-               - /var/run/docker.sock:/var/run/docker.sock
-             ports:
-               - 27778:27778
+            services:
+              mongo:
+                image: mongodb/atlas
+                privileged: true
+                command: |
+                  /bin/bash -c "atlas deployments setup --type local --port 27778 --bindIpAll --username root --password {your-password} --force && tail -f /dev/null"
+                volumes:
+                  - /var/run/docker.sock:/var/run/docker.sock
+                ports:
+                  - 27778:27778
+
+      #. Replace the password in the command on line 6 with your
+         password and save the file.
```

Comment:
Suggest the following in case the example or lines change.

```suggestion
      #. Replace the ``{your-password}`` variable in the example with your
         password and save the file.
```